_data/projects/Cuttle.yml

```
@@ -0,0 +1,14 @@
+name: Cuttle
+desc: A two-player battle card game for all ages.
```

Suggesting a revised description
```suggestion
desc: The open source, full-stack pvp implementation of Cuttle, the oldest battle card game played with a standard 52-card deck
```

---

Fixed locally and pushed it up. Also fixed the label name since I copied the wrong thing there, haha.